streaming/sender.c

```diff
@@ -725,4 +767,4 @@ void *rrdpush_sender_thread(void *ptr) {
 
     netdata_thread_cleanup_pop(1);
     return NULL;
-}
+}
```

Comment:
Please, add a newline at the end of this file to remove this warning.

Reply:
Done